src/gbo/GameBoyCore.js

```diff
@@ -4986,7 +4986,7 @@ GameBoyCore.prototype.recomputeDimension = function () {
 	//Cache some dimension info:
 	// this.onscreenWidth = this.canvas.width;
 	// this.onscreenHeight = this.canvas.height;
-	if (window && window.mozRequestAnimationFrame) {
+	if (window && window.mozRequestAnimationFrame || (navigator.userAgent.toLowerCase().indexOf("gecko") != -1 && navigator.userAgent.toLowerCase().indexOf("like gecko") == -1)) {
```

Comment:
Deep down in blink/webkit someone decided to not accelerate image scaling via CSS if the width/height were below 256 pixels in dimension.